.github/ISSUE_TEMPLATE.md

```diff
@@ -21,4 +24,4 @@
 
 ##### Complete output when you encounter the issue (if any)
 
-[INSERT OUTPUT HERE]
+```[INSERT OUTPUT HERE]```
```

Comment:
Sorry I wasn't clear, given that 3-backticks blocks are supposed to contain _multiple_ lines of code, I expected to have them in dedicated lines to let them wrap a _block_ (and not be inline), so more like this:

    ```
    [INSERT OUTPUT HERE]
    ```